src/encoded/types/file.py

```diff
@@ -176,6 +178,8 @@ def paired_with(self, root, request):
 
     @calculated_property(schema={
         "title": "Download URL",
+        "description": "The download path for S3 to obtain the actual file.",
```

Comment:
Not sure if this description would be more generic if the files moved to google cloud or azure.